src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java

```diff
@@ -471,4 +487,16 @@ public void siVisualizzaCorrettamenteLaPaginaNuovaDelega() {
         logger.info("Si visualizza la sezione Le Tue Deleghe");
         leTueDelegheSection.waitNuovaDelegaSection();
     }
+
+    @And("Si accetta la delega")
+    public void siAccettaLaDelega() {
+        BackgroundTest backgroundTest = new BackgroundTest();
+        backgroundTest.accettaDelegaPF();
+
+    }
+
+    @And("Si clicca sulle notifiche nel layout {string}" )
+    public void siCliccaSulleNotificheNelLayout(String nomeDelegante) {
+        destinatarioPage.clickButtonNotificheOnSideMenu(nomeDelegante);
+    }
```

Comment:
This is not clear, please adjust the full sentence of the feature step

Reply:
done